tests/e2e/README.md

```diff
@@ -0,0 +1,47 @@
+# E2E Tests
+
+End-To-End (E2E) tests for WordPress.
+
+
+## Running the tests
+
+The e2e tests require a production-like environment to run. By default, they will assume an environment is available at `http://localhost:8889`, with username=admin and password=password.
+
+You can launch the tests by running:
+
+```
```

Comment:
Is it worth mentioning that Docker needs to be running, and that `npm run env:start`/`npm run env:install` needs to be run prior to calling the actual test script? Without those prerequisites, the tests fail.

Reply:
I didn't want to mention a specific environment setup, because the tests can be run on any environment. For instance, using `@wordpress/env` package would also provide an env with the default specifications , but they can also be run on e.g. a MAMP setup, so Docker isn't strictly necessary. The important thing to know is we can pass a custom URL, username and password if the ones on our env don't match the defaults.

I guess we can link to the docs on how to set up a development env. I was assuming anyone trying to run these tests would already have that sorted, but perhaps it's best not to make any assumptions 😅

Reply:
Related question: if we do link to the docs, which ones should we link to? There's the [repo readme] with the Docker setup, and then there's [the handbook ](https://make.wordpress.org/core/handbook/tutorials/installing-a-local-server/) which has a bunch of different options.

Reply:
I think it is best to link to the repo readme.